core/services/eth/client.go

```diff
@@ -362,3 +363,10 @@ func (client *client) CallContext(ctx context.Context, result interface{}, metho
 	)
 	return client.RPCClient.CallContext(ctx, result, method, args...)
 }
+
+func (client *client) BatchCallContext(ctx context.Context, b []rpc.BatchElem) error {
+	logger.Debugw("eth.Client#BatchCall(...)",
+		"batchElems", b,
+	)
+	return client.RPCClient.BatchCallContext(ctx, b)
```

Comment:
this gets tested against geth and parity in the integration tests right?

Reply:
Yes I believe so